.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,36 @@
+name: Bug report
+description: Create a report to help us address an error.
+title: "[BUG] <bug description>"
+labels: [🚦 status: awaiting triage]
+assignees: nhcarrigan
+body:
+  - type: markdown
+    attributes:
+      value: |
+      # Bug report
+
+      ## Describe the bug
```

```suggestion
      value: '# Bug Report'
```
We shouldn't need the second line now as it's included in the input below.